### PR TITLE
Release 1.6.1

### PR DESCRIPTION
**Pull request by martijnbastiaan**
Are we doing release names yet? If so, I'd like to call this one _Whoopsie Daisy_.

Sorry @gergoerdi, @rowanG077, and thanks for noticing!

### CODE REVIEW

**Review by christiaanb — APPROVED** [review]

**Comment by martijnbastiaan** [comment]
It occurred to me that checking for the presence of `default: False` is more reliable than checking for the absence of `default: True`, so I've changed that. @alex-mckenna Would you mind checking my Bash?

**Comment by alex-mckenna** [comment]
So, how long until `1.6.2` :wink:

**Comment by DigitalBrains1** [comment]
We should commit the CI checks to `master` as well.

**Comment by christiaanb** [comment]
No, because on master we do want multiple-hidden enabled

File: .ci/setup.sh [hunk 1]
```diff
@@ -30,16 +30,29 @@ CI_COMMIT_TAG=${CI_COMMIT_TAG:-}
...
-if [[ ${tag_version} != "" && ${version} != ${tag_version} ]]; then
-    if [[ "${CI_COMMIT_TAG:0:1}" == "v" ]]; then
-        echo "Tag name and distribution's release number should match:"
-        echo "  Tag version:          ${CI_COMMIT_TAG}"
-        echo "  Distribution version: v${version}"
-        exit 1;
-    else
-        echo "\$CI_COMMIT_TAG should start with a 'v'. Found: ${CI_COMMIT_TAG}"
+# `tag_version` is set when a tag has been created on GitHub. We use this to
+# trigger a release pipeline (release to Snap / Hackage).
```
[the remark this answers is not in the document]
Comment (martijnbastiaan) [reply .ci/setup.sh:34]: That's correct @DigitalBrains1. We should commit it to `master` too. These commits should be forward ported:

* f75823ce6c4a8b59daa7d4ccde1177b4c3f26d57
* db4a1d991a082cdb9d2ebf8d3968f44904a6f86c

**Pull request merged**